### DISCUSSION TITLE
Retention Policy not working

### DISCUSSION THREAD

**Discussion by git-tec** [RocketChat/Rocket.Chat#29680] [Q&A]
We have enabled the following settings for the retention policy, but somehow no old chats are deleted in Omnichannel.

Do we need to set up some kind of cronjob? We have not found any information about this in the documentation.

![Retention Policy - settings2](https://github.com/RocketChat/Rocket.Chat/assets/136434874/052f88a2-9730-43ae-93c4-a8384fb9e33b)
![Retention Policy - settings](https://github.com/RocketChat/Rocket.Chat/assets/136434874/a680c2ed-7c33-4e08-9ab4-fcae929beb8f)

![Retention Policy - old chats](https://github.com/RocketChat/Rocket.Chat/assets/136434874/ac8dac68-2a3d-4f53-a224-8e3f7d2ea7a8)

**Comment by germebl** [comment]
Same issue(?) here. Maybe omnichannels didnt get cleaned up? Would be nice to get a answer from the team.
